### DISCUSSION TITLE
Recommendation of a IP camera with two way audio

### DISCUSSION THREAD

**Discussion by Oddsodz** [blakeblackshear/frigate#8123] [Ideas]
Hello all.

I am in the market for a camera for my front door.  And would love one that has 2 way audio.  As in one that I can speak to folks at my front door and they can speak back.

I understand that when Frigate is setup a certain way.  You can use the home assistant cards along with go2rtc to have two way conversations.  I think I have it all ready setup.  Just now need a camera that works great with Frigate go2rtc

So if anybody has a recommendations.  I would really appreciate it

Thanks for reading

Oddsodz.

**Comment by NickM-27** [comment]
are you looking for a turret camera or a doorbell camera?

**Comment by Oddsodz** [comment]
PoE turret/bullet would be best.

Power over Ethernet is mandatory due to the camera I plan to replace is also PoE

**Reply by NickM-27** [reply]
I personally have [Hikvision panoramic](https://www.bhphotovideo.com/c/product/1705511-REG/hikvision_colorvu_ds_2cd2387g2p_lsu_sl_8mp_network.html/?ap=y&ap=y&smp=y&smp=y&lsft=BI%3A514&gclid=EAIaIQobChMIovH34KTsgQMVPSytBh3QcgDbEAQYASABEgI77_D_BwE) and quite like it, but if you wanted a standard view camera I see [these options from dahua](https://us.dahuasecurity.com/product-selector/camera-selector/?product_tag=built-in-mic-speaker+bullet)
